### PR TITLE
Header of the web site bo done for evaluation.

### PR DESCRIPTION
**Pull request by evertonfxavier**

### CODE REVIEW

File: src/components/About/index.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const About = () => (
+  <Styled.AboutWrapper id="About">
+
+    <div>about</div>
+
+  </Styled.AboutWrapper>
+)
+
+export default About
```
Comment (clovisdasilvaneto) [inline src/components/About/index.jsx:12]: this is not part from the header, so it shouldn't be here

File: src/components/About/styled.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components"
+
+export const AboutWrapper = styled.div `
+    color: red;
+    max-width: 100%;
+    height: 890px;
+    background: #345cc6;
```
Comment (clovisdasilvaneto) [inline src/components/About/styled.js:7]: please create a theme provider and use it here. Take a look on styled component theme documentation

File: src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import * as Styled from './styled'
+import Logo from '../Logo/'
+import Menu from '../Menu/'
+import SocialMedia from '../SocialMedia'
+import scrollTo from 'gatsby-plugin-smoothscroll';
+import ScrollDown from '../ScrollDown/index'
+
+const Header = () => (
+  <Styled.SectionHeader id="Header">
+    {/* <Styled.HeaderShadow></Styled.HeaderShadow> */}
```
Comment (clovisdasilvaneto) [inline src/components/Header/index.jsx:11]: remove commented code.

File: src/components/Header/index.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import * as Styled from './styled'
+import Logo from '../Logo/'
+import Menu from '../Menu/'
+import SocialMedia from '../SocialMedia'
+import scrollTo from 'gatsby-plugin-smoothscroll';
+import ScrollDown from '../ScrollDown/index'
+
+const Header = () => (
+  <Styled.SectionHeader id="Header">
```
Comment (clovisdasilvaneto) [inline src/components/Header/index.jsx:10]: if you do not use this `id`, you don't need it.

File: src/components/Header/index.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import * as Styled from './styled'
+import Logo from '../Logo/'
+import Menu from '../Menu/'
+import SocialMedia from '../SocialMedia'
+import scrollTo from 'gatsby-plugin-smoothscroll';
+import ScrollDown from '../ScrollDown/index'
```
Comment (clovisdasilvaneto) [inline src/components/Header/index.jsx:7]: please put the dependencies on the top and the local files after that 👍

File: src/components/Header/styled.js [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+    background-attachment: fixed;
+    background-size: cover;
+    opacity: .9;
+    display: flex;
+    padding-bottom: 138px;
+
+    @font-face {
+        font-family: "montserrat-regular-webfont";
+        src: url(${montserrat});
+    }
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:19]: this should not be here, it should be in a globals file, take a look in useGlobals, from styled components documentation.

File: src/components/Header/styled.js [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+
+/*export const HeaderShadow = styled.div `
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 133%;
+    opacity: .6;
+    background-color: #000000;
+`*/
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:30]: remove commented code.

File: src/components/Header/styled.js [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+    max-width: 10%;
+    padding: 38px;
+    z-index: 2;
+`
+
+export const HeaderWrapper = styled.div `
+    
+
+    max-width: 55%;
+    font-family: "montserrat-bold-webfont";
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:42]: use it in a theme file

File: src/components/Header/styled.js [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+        h3{
+            line-height: 40px;
+            font-size: 14px;
+        }
+
+        h1{
+            font-size: 26px;
+            line-height: 50px;
+        }
+    }
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:105]: too many breakpoints, please try to make this layout adaptive.

File: src/components/Header/styled.js [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+    max-width: 450px;
+    display: flex;
+    justify-content: flex-start;
+    place-content: space-between;
+    padding-top: 50px;
+
+    button{
+        background: none;
+        color: #fff;
+        border: 2px solid #fff;
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:120]: use theme variables here.

File: src/components/Header/styled.js [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+        padding: 18px 46px;
+        text-transform: uppercase;
+        cursor: pointer;
+        transition: all 0.6s;
+
+        &:hover{
+            background: #fff;
+            color: #000;
```
Comment (clovisdasilvaneto) [inline src/components/Header/styled.js:128]: theme variables

File: src/components/Layout/layout.css [hunk 12]
```diff
@@ -0,0 +1,4 @@
+body{
+    margin: 0;
+    padding: 0;
+}
```
Comment (clovisdasilvaneto) [inline src/components/Layout/layout.css:4]: remove css file

File: src/components/Layout/index.jsx [hunk 13]
```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import "./layout.css"
```
Comment (clovisdasilvaneto) [inline src/components/Layout/index.jsx:2]: please use styled components instead css. If you want to set global styles, take a look on useGlobals from styled component

File: src/components/Logo/styled.js [hunk 14]
```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components";
+import montserrat from '../../fonts/montserrat/montserrat-semibold-webfont.woff2'
+
+export const LogoWrapper = styled.div `
+
+    @font-face {
+        font-family: "montserrat-bold-webfont";
+        src: url(${montserrat});
+    }
```
Comment (clovisdasilvaneto) [inline src/components/Logo/styled.js:9]: it should be in a globals file

**Comment by clovisdasilvaneto** [comment]
There's a lot of things to change here... Please take a look on the following subjects:

- Colors and fonts names should be set in a theme.js and use it via component props from styled components
- font-face should be only in a global file, it shouldn't be in a component style.
- Please take some time to look into styled-components docs. Read how to create global styles, theme, use theme variables inside a component and how to extend or subscribe a component css style.

File: src/components/About/index.jsx (hunk 1 above)
Comment (evertonfxavier) [reply src/components/About/index.jsx:12]: Done.

File: src/components/Header/index.jsx (hunk 3 above)
Comment (evertonfxavier) [reply src/components/Header/index.jsx:11]: Done.

File: src/components/Header/index.jsx (hunk 5 above)
Comment (evertonfxavier) [reply src/components/Header/index.jsx:7]: Done.

File: src/components/Header/styled.js (hunk 6 above)
Comment (evertonfxavier) [reply src/components/Header/styled.js:19]: Done.

File: src/components/Header/styled.js (hunk 7 above)
Comment (evertonfxavier) [reply src/components/Header/styled.js:30]: Done.

**Comment by clovisdasilvaneto** [comment]
@evertonfxavier I can not see any new commit, did you push it?

File: src/components/Layout/index.jsx (hunk 13 above)
Comment (evertonfxavier) [reply src/components/Layout/index.jsx:2]: Done.

File: src/components/Layout/layout.css (hunk 12 above)
Comment (evertonfxavier) [reply src/components/Layout/layout.css:4]: Done.

File: src/components/About/styled.js (hunk 2 above)
Comment (evertonfxavier) [reply src/components/About/styled.js]: Done.

File: src/components/Header/index.jsx (hunk 4 above)
Comment (evertonfxavier) [reply src/components/Header/index.jsx]: Done.

File: src/components/Header/styled.js (hunk 10 above)
Comment (evertonfxavier) [reply src/components/Header/styled.js]: Done.

File: src/components/Header/styled.js (hunk 11 above)
Comment (evertonfxavier) [reply src/components/Header/styled.js]: Done.

**Comment by clovisdasilvaneto** [comment]
@evertonfxavier I'm closing this PR. So many requested changes. Please work more on this branch by following the styled components docs https://styled-components.com/docs/basics

**Pull request closed without merging**